generators/languages/generator.ts

```diff
@@ -191,8 +187,12 @@ export default class LanguagesGenerator extends BaseApplicationGenerator<
     return this.asComposingTaskGroup({
       async bootstrap() {
         // Make sure generators languages callbacks are correctly initialized.
-        if ((this.jhipsterConfigWithDefaults as ClientConfig).clientFramework === 'angular') {
-          await this.dependsOnBootstrap('angular');
+        const { clientFramework, skipServer, backendType } = this.jhipsterConfigWithDefaults as ClientConfig;
+        if (clientFramework === 'angular' || clientFramework === 'react' || clientFramework === 'vue') {
```

Comment:
```suggestion
        if (clientFrameworkAny) {
```

Reply:
This would include blueprints.
We can catch the error and ignore or let the blueprint resolve.

Reply:
clientFrameworkAny doesn’t exist at this point.